tests/Behat/Element/Admin/Channel/ShowTheLowestPriceOfDiscountedProductsPriorToTheDiscountElement.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\PriceHistoryPlugin\Behat\Element\Admin\Channel;
+
+use FriendsOfBehat\PageObjectExtension\Element\Element;
+
+final class ShowTheLowestPriceOfDiscountedProductsPriorToTheDiscountElement extends Element implements ShowTheLowestPriceOfDiscountedProductsPriorToTheDiscountElementInterface
```

Comment:
Perhaps we could shorten this name to something similar to:
```suggestion
final class LowestPriceFlagElement extends Element implements LowestPriceFlagElementInterface
```
🤔